Tests/DVRTests/SessionTests.swift

```diff
@@ -269,4 +314,27 @@ class SessionTests: XCTestCase {
 
         wait(for: [firstExpectation, secondExpectation], timeout: 3.0)
     }
+        
+    func testEVSearch() throws {
```

Comment:
I think this one duplicates the coverage provided in `testTextPlaybackWithParams`, so probably we don't need the second test of the same type.

Reply:
Removed